cake/libs/configure.php

```diff
@@ -1035,7 +1035,11 @@ function __find($file, $recursive = true) {
 					require LIBS . 'folder.php';
 				}
 				$Folder =& new Folder();
-				$directories = $Folder->tree($path, array('.svn', '.git', 'CVS', 'tests', 'templates'), 'dir');
+				$ignorePaths = Configure::read('ignorePaths');
+				if(empty($ignorePaths)) {
```

Comment:
Ok without a configuration value it could be fixed like this:

On line 1038 in current 1.3 version (https://github.com/cakephp/cakephp/blob/1.3/cake/libs/configure.php#L1038)

Change:
$directories = $Folder->tree($path, array('.svn', '.git', 'CVS', 'tests', 'templates'), 'dir');

into this:
$directories = $Folder->tree($path, array('.svn', '.git', 'CVS', 'tests', 'templates', 'node_modules'), 'dir');